network-api/networkapi/wagtailpages/templatetags/special_template_page_tags.py

```diff
@@ -0,0 +1,24 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag('wagtailpages/blocks/tr_key_value_block.html', takes_context=True)
+def gettext(context, key):
+    page = context['page']
+
+    if 'value' in context:
+        del context['value']
```

Comment:
if there's multiple `gettext` on a page, this clears the previous `gettext`'s value binding